tap_hubspot/client.py

```diff
@@ -180,4 +289,29 @@ def get_url_params(
         params = super().get_url_params(context, next_page_token)
         if self.hs_properties:
             params["properties"] = ",".join(self.hs_properties)
+        if self._is_incremental_search(context):
+            return {}
```

Comment:
Should this be moved up to the top of the method's body? There's probably no reason to fill-in the `properties` param when doing an incremental search.